RetailCoder.VBE/ToDoItems/ToDoItem.cs

```diff
@@ -27,6 +28,9 @@ public class ToDoItem
         private readonly int _lineNumber;
         public int LineNumber { get { return _lineNumber; } }
```

Comment:
I didn't quite guet it. What you could have said is that `GetSelection` should be turned into a property and `LineNumber` would be binded as `Selection.StartLine`